code/modules/events/bureaucratic_error.dm

```diff
@@ -0,0 +1,18 @@
+/datum/event/bureaucratic_error
+	announceWhen = 1
+
+/datum/event/bureaucratic_error/announce()
+	GLOB.major_announcement.Announce("A recent bureaucratic error in the Human Resources Department may result in personnel shortages in some departments and redundant staffing in others.", "Paperwork Mishap Alert")
+
+/datum/event/bureaucratic_error/start()
+	var/list/jobs = SSjobs.occupations.Copy()
+	var/datum/job/overflow
+	var/errors
+	while(errors <= rand(1, 6))
+		var/random_change = pick(-1, 1)
+		overflow = pick_n_take(jobs)
+		if(!overflow.allow_bureaucratic_error)
+			continue
```

Comment:
would you make this filter out admin only as well just in case

Reply:
What do you mean by that? Setting the central.dm jobs to not allow errors isn't enough?

Reply:
```suggestion
		if(!overflow.allow_bureaucratic_error || overflow.admin_only)
			continue
```

Reply:
You resolved this but didn't make the change. Is this still necessary?

Reply:
I did commit this suggestion and removed the "allow_bureaucratic_error = TRUE" from admin roles since it was no longer necessary.

Reply:
Ah sorry I was looking at an outdated file version. Ignore me.